src/i18n/locale/en.json

```diff
@@ -311,6 +313,9 @@
   "components.MovieDetails.theatricalrelease": "Theatrical Release",
   "components.MovieDetails.tmdbuserscore": "TMDB User Score",
   "components.MovieDetails.viewfullcrew": "View Full Crew",
+  "components.MovieDetails.watchlistDeleted": "<strong>{title}</strong> Removed from watchlist  successfully!",
```

Comment:
Refer to my previous comment (extraction after fixing my previous comment should fix this)